dnsapi/dns_azion.sh

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env sh
+
+#
+#AZION_Username=""
+#AZION_Password=""
+#AZION_Token=""
+#AZION_ZoneID=""
+#
+
+AZION_Api="https://api.azionapi.net"
+
+########  Public functions ########
+
+# Usage: add  _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+# Used to add txt record
+dns_azion_add() {
+  fulldomain=$1
+  txtvalue=$2
+
+  AZION_Username="${AZION_Username:-$(_readaccountconf_mutable AZION_Username)}"
+  AZION_Password="${AZION_Password:-$(_readaccountconf_mutable AZION_Password)}"
+  AZION_Token="${AZION_Token:-$(_readaccountconf_mutable AZION_Token)}"
+  AZION_ZoneID="${AZION_ZoneID:-$(_readaccountconf_mutable AZION_ZoneID)}"
```

Comment:
please remove the "AZION_ZoneID",   we should not need it.
And we can not save it in the account conf.

The user may issue certs for different zones everytime.

we should  get the zone id by REST api every time.

Reply:
Removed